scripts/us_cdc/environmental_health_toxicology/precipitation/scripts/parse_precipitation_index.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+Author: Samantha Piekos @spiekos and Padma Gundapaneni @padma-g
+Date: 8/2/21
+Description: This script cleans up a csv file on county level
+precipitation data downloaded from the CDC.
+URL: https://data.cdc.gov/browse?category=Environmental+Health+%26+Toxicology
+@input_file   filepath to the original csv that needs to be cleaned
+@output_file  filepath to the csv to which the cleaned data is written
+python3 parse_precipitation_index.py input_file output_file
+'''
+
+import sys
+import pandas as pd
+
+def generate_dcid(fips):
+    list_dcids = []
+    for index, fip in fips.items():
+        fip = str(fip)
+        if len(fip) < 5:
+            fip = '0' + fip
```

Comment:
Instead of this check, why don't we just zfill(5) ?

Also, maybe call generate_dcid as generate_county_dcid so its obvious this is only for county?

Reply:
Missed this one. Nice catch! updated script and reprocessed files